cleaners/vuze.xml

```diff
@@ -34,11 +34,16 @@
   <label>Vuze/Azureus</label>
   <description>File transfer client</description>
   <var name="profile">
-    <value search="glob" os="windows">$APPDATA\Azureus</value>
+    <value search="glob" os="windows">$AppData\Azureus</value>
     <value search="glob" os="linux">~/.azureus</value>
   </var>
   <var name="programfiles">
-    <value search="glob" os="windows">$ProgramFiles\Vuze</value>
+    <!--
+    Windows itself changes %ProgramFiles% for 32-bit processes running in a 64-bit operating system as documented here:
+    https://docs.microsoft.com/en-us/windows/desktop/winprog64/wow64-implementation-details
+    ...so we use %SystemDrive%\Program Files\...
+    -->
+    <value search="glob" os="windows">%SystemDrive%\Program Files\Vuze</value>
```

Comment:
`ProgramW6432` is better than `%SystemDrive%\Program Files`

Reply:
But what's about W32 Support ???

Reply:
I don't understand your question. `ProgramW6432` points to `c:\program files\` regardless of whether BleachBit is 32-bit or 64-bit.

Reply:
I talk about the install path of Vuze on 32-bit systems... AFAIK is ProgramW6432 not available in Windows 32-bit...

Reply:
You mean 32-bit Windows, Vuze is installed in the `c:\program files` diretory and on 64-bit Windows it's installed in `c:\program files(x86)`? If so, then the regular  `%ProgramFiles%` would work as long as BleachBit is 32-bits.

Reply:
No, on W64 Vuze gets installed into `c:\program files` (64-bit version) and on W32 Vuze get installed into `c:\program files` (32-bit version).

Reply:
That seems odd, so it would help to have a comment in the XML